.babelrc

```diff
@@ -7,14 +7,10 @@
       }
     ]
   ],
+  "plugins": "transform-runtime",
```

Comment:
Should this be an array?

Reply:
@deini it can be an array or a string. Happy to change it to an array.